mantidproject/mantid
# Engineering GUI save Nexus file with all the other output after a focus

Previously, when running a focus from the ED GUI, the focused `.nxs` file wasn't saved in the output directory, it was saved in the MantidInstall directory. This PR makes the file easier to find by including it with all the other focus output.

**To test:**

Run a focus. To do this, go to the GUI (Interfaces -> Diffraction -> Engineering Diffraction).

You'll need to run a calibration first - go to the Calibration tab, and under **Make new calibration**, enter 236516 in **Vanadium #** and 241391 in **Calibration Sample #**. Click **Calibrate** (this might take a while!).

Now go to the Focus tab, and select [this file](https://github.com/mantidproject/mantid/files/1964232/ENGINX00281188.zip). Pick whichever banks you want to focus, and click **Focus**. This also might take a while. Go to your output directory (on Windows `C:\EnginX_Mantid\User\<rb_number>\Focus`, on Linux `~/EnginX_Mantid/User/<rb_number>/Focus`) and make sure that there is a new Nexus file there for each bank that you selected to focus. Make sure these look sensible by opening them up in MantidPlot.

Fixes #22352 
Once this is closed go and tick off the relevant item in #21022

**Release Notes** 
[Here](https://github.com/mantidproject/mantid/commit/1d04cc903e9db4732fc3ebec3be50a34c1d01973)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
